dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py

```diff
@@ -0,0 +1,122 @@
+import time
+
+from python_fb_page_insights_client import (
+    FBPageInsight,
+    PageWebInsightData,
+    PostsWebInsightData,
+    DatePreset,
+    Period,
+)
+from pydantic import BaseSettings
+from typing import List, Dict, Union
+
+from google.cloud import bigquery
```

Comment:
btw, I forgot to mention that we should use [bigquery hook](https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/cloud/hooks/bigquery/index.html) or [bigquery operator](https://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/bigquery.html). It's a better way to handle connections in Airflow

Reply:
Due to the running out of time and no other file in this repo already starts to use `bigquery hook` or `bigquery operator`, is it good to try these two in another PR?